5-April-22/genesis.md

```diff
@@ -5,6 +5,26 @@ It's time for the phoenix genesis.
 Download it via the links below, and _ensure the hash matches_.
 
 ### Download
+
+Get from `EZ Staking`:
+
+- https://snapshots.ezstaking.io/juno-phoenix-genesis.tar.gz
+- https://static.ezstaking.io/juno-phoenix-genesis.tar.gz
+- https://static-2.ezstaking.io/juno-phoenix-genesis.tar.gz
+- https://static-3.ezstaking.io/juno-phoenix-genesis.tar.gz
+- https://ezstaking.io/juno-phoenix-genesis.tar.gz
+
+```sh
+wget https://snapshots.ezstaking.io/juno-phoenix-genesis.tar.gz
+tar -xvf juno-phoenix-genesis.tar.gz -C $HOME/.juno/config
+# check chain is juno-1, genesis time is correct & initial block is 2578099
+# note if using zsh that you may need to break this up, and run steps individually
+# i.e. cat $HOME/juno/config/genesis.json | jq '.chain_id'
+cat $HOME/.juno/config/genesis.json | jq '"Genesis Time: " + .genesis_time + " — Chain ID: " + .chain_id + " - Initial Height: " + .initial_height'
+# check hash - returns de3db1f58f612b0706eef5bee8f83fa46b9dcd1159efe582c33f3d1907434f7a
```

Comment:
hash needs updating to 36753f4f726aec8816dc38fa2453e4e70f8f4fccf0f0458e71ee37f362d758f1 in this comment - my mistake sorry

Reply:
Done